test/ts/suspense-test.tsx

```diff
@@ -0,0 +1,38 @@
+import "mocha";
+import { expect } from "chai";
+import {
+	createElement,
+	Component,
+        Suspense,
+        lazy,
+        AnyComponent
+} from "../../src/";
+import { IsLazyComponent } from './lazy';
+
+const FallBack = () => <div>Still working...</div>;
+/**
+ * Have to moch dynamic import as import() throws a syntax error in the test runner
```

Comment:
"mock" * 🙃

Reply:
Haha whoops, going to go through things for a cleanup tomorrow.

Reply:
Fixed!